app/assets/stylesheets/main.css.scss

```diff
@@ -52,3 +52,6 @@ $compatibility-high-color: #84C93F;
 @import 'styles/pages/home';
 @import 'styles/pages/styles';
 
+// Import modular CSS
+@import "modules/**/**/*";
+@import "modules/**/**/**/*";
```

Comment:
Thanks for the PR! 

I may not understand the intricacies of the import statement here, does this mean that more nested objects will be imported after the parent objects?